app/views/plants/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>Update Plant Information</h1>
+<%= form_for @plant do |f| %>
+  <% if @errors %>
+    <ul>
+      <div class="flash">
```

Comment:
Since we have the flash notices in our application layout, do we need it on this view page?